src/replication_job.js

```diff
@@ -36,6 +37,10 @@ async function main() {
 
   learningContentReplicationQueue.process(async function() {
     await steps.importLearningContent(configuration);
+    notificationQueue.add({}, jobOptions);
+  });
+
+  notificationQueue.process(async function() {
```

Comment:
Question: Pourquoi le log 'Import and enrichment done' a été déplacé dans l'étape de notification. Ce log concerne bien l'étape d'avant, si jamais la notification a été échoué l'import et l'enrichissement sera bien effectué. 
Proposition: Rajouter un log spécifique à l'étape de notification

Reply:
Il y a un log spécifique, mais j'ai déplacé ce log pour des raisons historique. Le monitor datadog se base la dessus pour notifier la fin de la réplication, et j'aimerais qu'on soit notifier après la notification